<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApbsDialog</class>
 <widget class="QDialog" name="ApbsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>APBS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Structure Input File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="generateFromPdbButton">
          <property name="text">
           <string>Generate Input From PDB</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="pdbFileLabel">
            <property name="text">
             <string>PDB File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pdbFileLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="openPdbFileButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Force Field:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="forceFieldComboBox">
            <item>
             <property name="text">
              <string>AMBER</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CHARMM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PARSE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TYL06</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PEOEPB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SWANSON</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runPdb2PqrButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Run PDB2PQR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="loadFromPqrButton">
          <property name="text">
           <string>Load Existing PQR File</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pqrFileLabel">
            <property name="text">
             <string>PQR File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pqrFileLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="openPqrFileButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Avogadro::QtGui::InputGeneratorWidget" name="inputGenerator" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Avogadro::QtGui::InputGeneratorWidget</class>
   <extends>QWidget</extends>
   <header location="global">avogadro/qtgui/inputgeneratorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
